packages/css/src/components/label-group/label-group.template.ts

```diff
@@ -1,10 +1,11 @@
-// Keep in sync packages\core\src\components\label-group\label-group.template.ts (https://github.com/dso-toolkit/dso-toolkit/issues/1438)
 import { LabelGroup } from '@dso-toolkit/sources';
 import { html } from 'lit-html';
-import { labelTemplate } from '../label/label.template';
+import { labelTemplate } from '@dso-toolkit/core/src/components/label/label.template';
 
 export function labelGroupTemplate({ labels }: LabelGroup) {
-  return html`<div class="dso-label-group">
-    ${labels.map(labels => labelTemplate(labels))}
-  </div>`;
+  return html`<ul class="dso-label-group">
```

Comment:
Ik denk dat ik liever zie dat de `<ul>` _in_ de `.dso-label-group` komt te staan.

Mochten we ooit een Web Component/custom element implementatie van Label Group maken komen we niet weg met `<dso-label-group>`. Maar als de `ul` IN het component staat wel:

```html
<div class="dso-label-group">
  <ul>
    <li>[..]</li>
  </ul>
</div>

<dso-label-group>
  <ul>
    <li>[..]</li>
  </ul>
</dso-label-group>
```

Reply:
Check